vite.config.js

```diff
@@ -9,6 +9,9 @@ export default defineConfig({
       checker: 'vue-tsc',
       ignoreSourceErrors: true,
     },
+    fakeTimers: {
+      enable: true,
+    },
```

Comment:
what does this do? There's a beforeEach and afterEach that enables and disabled fake timers. Does this make those not necessary because it just always does that?